### PR TITLE
bump Default1PBlocking to 100% disabled on release channel

### PR DESCRIPTION
**Pull request by antonok-edm**

### CODE REVIEW

**Review by pes10k — APPROVED** [review]
lgtm

**Review by iefremov — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` at `100%` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
